tests/suites/0_stateless/20+_others/20_0012_privilege_access.result

```diff
@@ -19,7 +19,7 @@ test -- select
 1
 1
 1
-Error: APIError: ResponseError with 1063: Permission denied, privilege [Select] is required on 'default'.'default2'.'v_t20_0012' for user 'test-user'@'%' with roles [public,test-role1,test-role2]
+1
```

Comment:
Looks it's time to spill out a new dir logic tests for RBAC from `20+_others`, we can do in later PR  :)